bugbug/repository.py

```diff
@@ -411,131 +352,99 @@ def download_commits(repo_dir, date_from):
         path: "::".join(component) for path, component in path_to_component.items()
     }
 
-    global components_touched_prev
-    global components_touched_prev_90_days
+    first_pushdate = commits[0].pushdate
 
-    global files_touched_prev
-    global files_touched_prev_90_days
+    experiences = defaultdict(lambda: defaultdict(lambda: defaultdict(int)))
+    complex_experiences = defaultdict(lambda: defaultdict(lambda: defaultdict(set)))
```

Comment:
nit: doc what complex experiences are... it's unclear for me.

Reply:
It's for the case where you need to have a set of commits.
E.g. say commit A is modifying "dom" and "apps", commit B is modifying "apps", commit C is modifying "dom" and "apps".

For commit C, you should have that the number of previous commits touching the same dirs is 2 (A and B), but if you just use the sum you will have 3 (A twice + B)

Reply:
Fixed with 90bf7cf006ec218059e1a298f452c4314c5246aa.